src/voodoo/spellbook/spells/stoneskin.ts

```diff
@@ -35,7 +35,7 @@ export const stoneskin: SpellFunction = async (voodoo, accountId, upgradeConfigs
     }
   });
 
-  const multiplier = 1 + attributes.intensify / 100;
+  const multiplier = attributes.intensify / 100;
```

Comment:
Implementing mdingena's suggestions
```suggestion
  const bonus = attributes.intensify / 100;
```